src/include/sof/lib/clk.h

```diff
@@ -38,6 +38,7 @@ struct clock_info {
 	const struct freq_table *freqs;
 	uint32_t default_freq_idx;
 	uint32_t current_freq_idx;
+	uint32_t waiti_freq_idx;
```

Comment:
Ok, I'm reading the comments about things becoming more complex so I think we can simplify by adding an inline comment to each "idx" and "notification" member in this struct. This should start to make things simpler for all reviewers to follow.

Reply:
Should this index be a common parameter? Seems to be specific for cAVS platforms and left uninitialized/unused for others.